regress/connect-unix.sh

```diff
@@ -0,0 +1,45 @@
+tid="simple connect"
+
+SCRIPT_DIR=`dirname $0`
+
+start_sshd_unix()
+{
+	${SUDO} env NC=${NC} SSHD=${SSHD} OBJ=${OBJ} ${SCRIPT_DIR}/sshd-unix.sh&
```

Comment:
One reason for the CI error `FATAL: no sshd.socket created` is because if a timing issue. Here we run the whole script `sshd-unix.sh` in the background, which first cleans up `sshd.socket` before starting netcat to create a new file. This creates a timing window where the wait for creating of `sshd.socket` finishes too early. I suggest the following change:
```diff
modified   regress/connect-unix.sh
@@ -4,7 +4,7 @@ SCRIPT_DIR=`dirname $0`
 
 start_sshd_unix()
 {
-	${SUDO} env NC=${NC} SSHD=${SSHD} OBJ=${OBJ} ${SCRIPT_DIR}/sshd-unix.sh&
+	${SUDO} env NC=${NC} SSHD=${SSHD} OBJ=${OBJ} ${SCRIPT_DIR}/sshd-unix.sh
 
 	trace "wait for sshd-unix"
 	i=0;
modified   regress/sshd-unix.sh
@@ -7,7 +7,7 @@ run_sshd_unix() {
 	rm -f ${OBJ}/sshd.fifo && mkfifo ${OBJ}/sshd.fifo
 	rm -f ${OBJ}/sshd.socket
 	touch ${OBJ}/sshd-unix.log ; chmod 0644 ${OBJ}/sshd-unix.log
-	cat ${OBJ}/sshd.fifo | ${SSHD} -i -f ${OBJ}/sshd_config "$@" -E ${OBJ}/sshd-unix.log | ${NC} -l -U ${OBJ}/sshd.socket > ${OBJ}/sshd.fifo
+	cat ${OBJ}/sshd.fifo | ${SSHD} -i -f ${OBJ}/sshd_config "$@" -E ${OBJ}/sshd-unix.log | ${NC} -l -U ${OBJ}/sshd.socket > ${OBJ}/sshd.fifo &
 }
 
 run_sshd_unix
```

I haven't been able to run the test suite in full on my own machine, so I can't tell if this fixes the CI error. But at least it should be more logically correct.

Reply:
Thanks! I just pushed a change with that fix.